### PR TITLE
Add docker-ipam-tools image for create-ea-defs

### PR DESCRIPTION
**Pull request by ishantt**
This patch adds docker-ipam-tools image to run create-ea-defs
tool.
Use Alpine 3.5 version for the base image.

### CODE REVIEW

File: common/config.go [hunk 1]
```diff
@@ -1,57 +1,72 @@
...
 )
 
 const configFileDir = "/etc/infoblox"
 
+type Config interface {
+	LoadFromConfFile() error
+	LoadConfig() error
+}
+
 type GridConfig struct {
```
Comment (yuewko) [inline common/config.go:19]: Need documentation update as a result of these changes.

**Review by yuewko — COMMENTED** [review]
Looks good.

Just one comment inline.

File: common/config.go (hunk 1 above)
Comment (ishantt) [reply common/config.go:19]: Thanks for reviewing Ko! Yes I will update the documentation in a separate patch. I have lots left to document :)

Comment (yuewko) [reply common/config.go:19]: I think you should at least update the document in this patch just to reflect the new config parameter names. (just change the '_' to '-'

**Review by yuewko — APPROVED** [review]
LGTM!

**Pull request merged**